source/osmaxx/templates/base.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+{% load staticfiles %}
+
+<html>
+<head lang="en">
+    <meta charset="UTF-8">
+    <title>Osmaxx</title>
+    <style>
+        body { font-family: Arial, sans-serif; }
+    </style>
+
+    <link rel="stylesheet" href="http://cdn.leafletjs.com/leaflet-0.7.3/leaflet.css" />
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap.min.css" />
+    <link rel="stylesheet" type="text/css" media="screen" href="https://silviomoreto.github.io/bootstrap-select/bootstrap-select.min.css" />
+    {% block extra_head %}{% endblock %}
+    <link rel='stylesheet' type='text/css' href='{% static "excerptexport/stylesheets/main.css" %}' />
+</head>
+<body>
+{% block body %}
+
+    {% if messages %}
```

Comment:
Maybe it makes sense to move this account specific template parts before header to a partial and use an include?